### PR TITLE
info command

### PR DESCRIPTION
**Pull request by hkubus**

### CODE REVIEW

**Review by Nyameliaaaa — CHANGES_REQUESTED** [review]
you should get the tags using IDs. also i think its kinda duplicated

File: src/components/info.ts [hunk 1]
```diff
@@ -0,0 +1,79 @@
+import FluorineClient from '@classes/Client';
+import Embed from '@classes/Embed';
+import { ButtonInteraction, MessageActionRow, MessageButton } from 'discord.js';
+export const authorOnly = true;
+export async function run(client: FluorineClient, interaction: ButtonInteraction, value: string) {
+    let embed: Embed;
```
Comment (SuperchupuDev) [inline src/components/info.ts]:
```suggestion
    const embed: = new Embed(client, interaction.locale);
```

File: src/components/info.ts [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+            .setCustomId(`info:${interaction.user.id}:info`)
+            .setStyle('PRIMARY'),
+        new MessageButton()
+            .setLabel(client.i18n.t('INFO_STATS', { lng: interaction.locale }))
+            .setDisabled(value === 'stats')
+            .setCustomId(`info:${interaction.user.id}:stats`)
+            .setStyle('PRIMARY')
+    ]);
+    if (value === 'info') {
+        embed = new Embed(client, interaction.locale)
```
Comment (SuperchupuDev) [inline src/components/info.ts]:
```suggestion
        embed
```

File: src/components/info.ts [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+                value: 'en-US, pl',
+                inline: true
+            })
+            .addLocaleField({
+                name: 'INFO_VERSION',
+                value: client.version,
+                inline: true
+            });
+    } else {
+        embed = new Embed(client, interaction.locale)
```
Comment (SuperchupuDev) [inline src/components/info.ts]:
```suggestion
        embed
```

**Review by SuperchupuDev — CHANGES_REQUESTED** [review]
this will probably mess up eslint

**Review by SuperchupuDev — APPROVED** [review]

**Review by Nyameliaaaa — APPROVED** [review]
ok fine ill polish patch this into a better command

**Pull request merged**